apps/portal/lib/graphql/services/Firebase.service.ts

```diff
@@ -0,0 +1,49 @@
+import { PrismaClient } from '@prisma/client';
+import { singleton } from 'tsyringe';
+const { initializeApp, getApps, applicationDefault, cert } = require('firebase-admin/app');
+const { getFirestore, Timestamp, FieldValue } = require('firebase-admin/firestore');
+const serviceAccount = require('C:/UTD/Senior Year/Spring Semester 2022/Projects/try-acm/practice-5d375-firebase-adminsdk-tbfef-003353b527.json');
+require('dotenv').config();
+
+@singleton()
+export default class Firebase {
+  db: any;
```

Comment:
Can you change this to anything else other than `any`?

Reply:
Nope - getFirestore - when hovered over - gives value of any

Reply:
Ok. I'll resolve this conversation then.